recaptcha_enterprise/demosite/src/main/resources/templates/store.html

```diff
@@ -58,35 +58,30 @@
     <link href="/styles/global.css" rel="stylesheet" />
     <script type="module" src="/scripts/component.js"></script>
     <script type="text/javascript" src="/scripts/global.js"></script>
-    <!-- ATTENTION: reCAPTCHA Example (Part 1) Starts -->
-    <!-- SEE: https://cloud.google.com/recaptcha-enterprise/docs/instrument-web-pages#user-action -->
+    <!-- ATTENTION: reCAPTCHA Example (Client Part 1/2) Starts -->
+    <!-- See: https://cloud.google.com/recaptcha-enterprise/docs/instrument-web-pages#user-action -->
     <script th:src="@{https://www.google.com/recaptcha/enterprise.js(render=${site_key})}"></script>
     <script type="text/javascript">
       function handleClick(event) {
         event.preventDefault();
         grecaptcha.enterprise.ready(async () => {
           const token = await grecaptcha.enterprise.execute(
               "[[${site_key}]]",
-            {
-              action: "check_out",
-            }
+              {
+                action: "check_out",
+              }
           );
-          // SEE: https://cloud.google.com/recaptcha-enterprise/docs/create-assessment
-          const score = await createAssessment({
-            action: "check_out",
-            token,
-          });
-          useAssessment(score);
+          await store({token});
```

Comment:
Can we call this `checkout` instead of store to be consistent?

Reply:
Store is the webpage name and the "checkout" refers to an action. There can be multiple actions in a webpage and hence prefer keeping it as such.